prime-router/metadata/fhir_mapping/hl7/resource/Patient.yml

```diff
@@ -265,6 +265,39 @@ birthDate:
   valueOf: PID.7
   expressionType: HL7Spec
 
+_birthDate:
+  condition: $pid7 NOT_NULL
+  vars:
+    pid7: PID.7
+    dateTimeIn: PID.7, GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)
+    isTime: PID.7, GeneralUtils.dateTimeWithZoneId(isTime,ZONEID).length() > 10
```

Comment:
This is good to know that this works, I think we had opened a bug, but might have been looking for a an operator rather than a Java solution.

I think this means we can remove this [now](https://github.com/CDCgov/prime-reportstream/blob/544cc8ccb5120c35653a21763e33a5835f502800/prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt#L335)?  Also, is there reason we need both extensions?  It looks like it would contain the same info.


Maybe just a comment as to the difference?

Reply:
Nevermind this makes sense now for places with less precision